packages/kbn-securitysolution-io-ts-list-types/src/request/summary_exception_list_schema/index.test.ts

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { left } from 'fp-ts/lib/Either';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { exactCheck, foldLeftRight, getPaths } from '@kbn/securitysolution-io-ts-utils';
+
+import { getSummaryExceptionListSchemaMock } from './index.mock';
+import { SummaryExceptionListSchema, summaryExceptionListSchema } from '.';
+
+describe('summary_exception_list_schema', () => {
+  test('it should validate a typical exception list request', () => {
+    const payload = getSummaryExceptionListSchemaMock();
+    const decoded = summaryExceptionListSchema.decode(payload);
+    const checked = exactCheck(payload, decoded);
+    const message = pipe(checked, foldLeftRight);
```

Comment:
Just curious, what is the reason to call a single function using `pipe`?
```js
const message = pipe(checked, foldLeftRight);
```
IMO, it looks fancy but makes things a little bit harder to read. Wouldn't it be simpler to call it without the pipe?
```js
const message = foldLeftRight(checked);
```

Reply:
Yeah, that's right. I just did copy&paste from the other tests in the package. I'll do the change but take into account that other tests like this `packages/kbn-securitysolution-io-ts-list-types/src/request/read_exception_list_schema/index.test.ts` are using pipe.